src/source/PeerConnection/PeerConnection.c

```diff
@@ -1045,6 +1045,25 @@ STATUS restartIce(PRtcPeerConnection pPeerConnection)
     return retStatus;
 }
 
+STATUS closePeerConnection(PRtcPeerConnection pPeerConnection)
+{
+    ENTERS();
+    STATUS retStatus = STATUS_SUCCESS;
+    PKvsPeerConnection pKvsPeerConnection = (PKvsPeerConnection) pPeerConnection;
+
+    CHK(pKvsPeerConnection != NULL, STATUS_NULL_ARG);
+
+    CHK_STATUS(dtlsSessionShutdown(pKvsPeerConnection->pDtlsSession));
```

Comment:
Should this be CHK_LOG_ERR?

Reply:
sry, iceAgentShutdown should be CHK_STATUS